services/app-web/src/pages/StateSubmission/ReviewSubmit/V2/ReviewSubmit/RateDetailsSummarySectionV2.tsx

```diff
@@ -294,29 +294,47 @@ export const RateDetailsSummarySectionV2 = ({
                               </h3>
                               <dl>
                                   <DoubleColumnGrid>
-                                      {rateFormData.deprecatedRateProgramIDs
+                                      {((rateFormData.deprecatedRateProgramIDs
                                           .length > 0 &&
-                                          isSubmittedOrCMSUser && (
-                                              <DataDetail
-                                                  id="historicRatePrograms"
-                                                  label="Programs this rate certification covers"
-                                                  explainMissingData={
-                                                      false // this is a deprecated field, we never need to explain if its missing
-                                                  }
-                                                  children={ratePrograms(
-                                                      rate,
-                                                      true
-                                                  )}
-                                              />
-                                          )}
-                                      <DataDetail
-                                          id="ratePrograms"
-                                          label="Rates this rate certification covers"
-                                          explainMissingData={
-                                              explainMissingData
-                                          }
-                                          children={ratePrograms(rate, false)}
-                                      />
+                                          isSubmittedOrCMSUser) ||
+                                          (rateFormData.deprecatedRateProgramIDs
```

Comment:
nit: this feels hard to grok with so many chained conditionals nested in the jsx code - could we make this a variable higher up similar to `isSubmittedOrCMSUser` ? I think this is `isLockedAndOnlyHasHistoricRates` or something like that